docs/content/dependencies.md

```diff
@@ -17,13 +17,14 @@ Here is a [full example][example] of a Porter manifest that uses dependencies.
 
 In the manifest, add entries for each dependency of your bundle. The `name` field takes a short name for the dependent bundle that
 you will use to reference the dependent bundle elsewhere in the bundle. For example you can reference the dependent bundle's
-outputs via `{{ bundle.dependencies.NAME.outputs }}`.  The `reference` field takes the bundle reference of the dependency.
+outputs via `{{ bundle.dependencies.requires.NAME.outputs }}`.  The `reference` field takes the bundle reference of the dependency.
```

Comment:
I don't think that we want to change the template variables. Another issue for v1 (#1571) is to move the variable for dependencies out of bundle, under "installation", to make it more clear that it's a runtime variable.